appveyor.yml

```diff
@@ -52,6 +52,7 @@ matrix:
     - OTP_VERSION: 21.2
 
 init:
+  - ps: Update-AppveyorBuild -Version "$env:appveyor_build_version-$($env:appveyor_repo_commit.substring(0,7))"
```

Comment:
In the specified value, after the `-`, what is that extra `$(...)` i.e. why `$($env:appveyor_repo_commit.substring(0,7))` and not `$env:appveyor_repo_commit.substring(0,7)`?

Reply:
This is a sub-expression. Making sure that variables are properly scoped. Similar to bash actually.